net-misc/teamviewer/teamviewer-15.20.6.ebuild

```diff
@@ -0,0 +1,166 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_MAJOR="$(ver_cut 1)"
+MY_P="${PN}${MY_MAJOR}"
+inherit desktop optfeature systemd xdg
+
+DESCRIPTION="All-In-One Solution for Remote Access and Support over the Internet"
+HOMEPAGE="https://www.teamviewer.com"
+SRC_URI="amd64? ( https://dl.tvcdn.de/download/linux/version_${MY_MAJOR}x/${PN}_${PV}_amd64.tar.xz )
+		arm? ( https://dl.tvcdn.de/download/linux/version_${MY_MAJOR}x/${PN}_${PV}_armhf.tar.xz )
+		x86? ( https://dl.tvcdn.de/download/linux/version_${MY_MAJOR}x/${PN}_${PV}_i386.tar.xz )"
+S="${WORKDIR}"/teamviewer
+
+LICENSE="TeamViewer MIT"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+
+# Unpack will fail without app-arch/xz-utils[extra-filters], bug #798027
+BDEPEND="app-arch/xz-utils[extra-filters]"
+
+RDEPEND="
+	dev-libs/nss
+	dev-libs/nspr
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/libglvnd[X]
+	sys-apps/dbus
+	sys-apps/util-linux
+	sys-libs/zlib:0/1[minizip]
+	!sys-auth/consolekit
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libxcb
+	x11-libs/libxkbcommon[X]
+	x11-libs/libXcomposite
+	x11-libs/libXcursor
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrandr
+	x11-libs/libXrender
+	x11-libs/libXScrnSaver
+	x11-libs/libXi
+	x11-libs/libXtst
+	x11-libs/libX11
+	x11-libs/xcb-util-image
+	x11-libs/xcb-util-keysyms
+	x11-libs/xcb-util-renderutil
+	x11-libs/xcb-util-wm
+"
+
+# For consolekit incompatibility see https://forums.gentoo.org/viewtopic-p-8332956.html#8332956
+
+RESTRICT="bindist mirror"
```

Comment:
Oh missed that, usually RESTRICT is put together with the LICENSE/KEYWORDS block to match skel.ebuild ordering rather than down here.

Reply:
yes. fixed.